.ci/run_container.sh

```diff
@@ -15,18 +15,17 @@ fi
 "${CONTAINER_RUNTIME}" run --rm --detach --name "pulp" --volume "${BASEPATH}/settings:/etc/pulp" --publish "8080:80" pulp/pulp-fedora31
 
 echo "Wait for pulp to start."
-for i in $(seq 10)
+for _ in $(seq 10)
 do
   sleep 3
   if curl --fail http://localhost:8080/pulp/api/v3/status/ > /dev/null 2>&1
   then
     echo "SUCCESS."
     break
   fi
-  echo -n "."
 done
 
-trap "${CONTAINER_RUNTIME} stop pulp" EXIT
+trap '${CONTAINER_RUNTIME} stop pulp' EXIT
```

Comment:
Was this really problematic? We knew the CONTAINER_RUNTIME at installing the trap. This makes it lazy evaluated, i think.